xclim/data/fr.json

```
@@ -705,23 +704,23 @@
     "abstract": "Calcul de la magnitude et la direction de la vitesse du vent à partir des deux composantes ouest-est et sud-nord."
   },
   "WIND_SPEED_FROM_VECTOR.sfcWind": {
-    "long_name": "Vitesse du vent de surface",
+    "long_name": "Vitesse du vent près de la surface",
     "description": "Magnitude de la vitesse du vent calculée à partir des deux composantes uas et vas."
   },
   "WIND_SPEED_FROM_VECTOR.sfcWindfromdir": {
-    "long_name": "Direction de provenance du vent de surface",
+    "long_name": "Vent près de la surface en provenance d'une certaine direction",
     "description": "Direction de provenance du vent calculée à partir des deux composantes uas et vas. Les vents du nord ont une direction de 360° et les vents de moins de {calm_wind_thresh} ont une direction de 0°."
   },
   "WIND_VECTOR_FROM_SPEED": {
     "title": "Vitesse cartésienne du vent à partir de la magnitude et direction de provenance.",
```

```suggestion
    "title": "Vitesse cartésienne du vent à partir de la magnitude et de la direction.",
```
Je pense qu'on peut y aller avec juste "direction", ici.